microservices/Order.py

```diff
@@ -0,0 +1,132 @@
+from flasgger import swag_from
+from flask import Blueprint, request, session
+from flask_login import login_required, login_user, logout_user
+from werkzeug.security import check_password_hash
+from flask import Blueprint, request, make_response, jsonify
+from flask.views import MethodView
+from project.server import bcrypt, db
+from project.server.models import User, BlacklistToken
+
+@app.route('/')
+@app.route('/order/')
+def showorders():
+    # query db and assign to orders variable
+    orders = session.query(order).order_by(order.id.asc()).all()
+
+    # Check if user credential match data in login_session
+    # Check if username data in login_session
+    if 'username' not in login_session:
+        # This page will redirect user to Google login page
+        return render_template('publicorders.html', orders = orders, title = "orders")
+    else:
+        # return "This page will show all orders"
+        return render_template('orders.html', orders = orders, title = "orders")
+
+
+@app.route('/order/new/', methods = ['GET', 'POST'])

```

Comment:
`/order/new/` should probably just be a POST request

`/order/{id}` should be a GET where you pass in some sort of id